h2o-logger/src/main/java/logging/ConsoleLogger.java

```diff
@@ -0,0 +1,43 @@
+package logging;
+
+import java.io.PrintStream;
+
+public class ConsoleLogger implements ILogger {
```

Comment:
do we really need this? the default slfj implementation I think does pretty much the same, so I would just use slf4j in genmodel and be done with it, @michalkurka WDYT?

Reply:
in http://www.slf4j.org/manual.html there is "if no binding is found on the class path, then slf4j-api will default to a no-operation implementation discarding all log requests."

Reply:
oh right, thats a bit of a conundrum, then we probably need this class